cpu/esp32/startup.c

```diff
@@ -342,7 +354,12 @@ static NORETURN void IRAM system_init (void)
     esp_event_handler_init();
 
     /* starting RIOT */
-    printf("Starting RIOT kernel on PRO cpu\n");
+#ifdef MODULE_ESP_LOG_STARTUP
+    LOG_STARTUP("Starting RIOT kernel on PRO cpu\n");
+    uart_tx_wait_idle(CONFIG_CONSOLE_UART_NUM);
+#else
+    puts("");
```

Comment:
Could the else part be simply removed ?

Reply:
It could be removed but it seems a bit strange when RIOT's welcome message comes directly below the ROM bootloader messages which cannot be disabled at all. It simply looks better with the additional line break.

Reply:
ok